### DISCUSSION TITLE
Need help in disabling Top Sites (explore-sites flag) option

### DISCUSSION THREAD

**Discussion by lordarcadius** [bromite/bromite#2062] [Q&A]
Hey, I was building a chromium-based browser and I want to disable the "Top Sites" feature which can be disabled by the "explore-sites" flag. I want to disable this flag by default or want to disable this feature itself. I am not really able to figure it out. I saw that bromite doesn't have this enabled. Please help me to disable it as well.

**Comment by uazo** [comment]
> I saw that bromite doesn't have this enabled.

I don't know exactly what you are referring to, but this is the newtab page patch used in bromite:
https://github.com/bromite/bromite/blob/master/build/patches/Restore-Simplified-NTP-launch.patch

> Please help me to disable it as well.

I'm sorry, but this I think will not be possible

**Comment by csagan5** [comment]
> Hey, I was building a chromium-based browser and I want to disable the "Top Sites" feature which can be disabled by the "explore-sites" flag.

These discussions are for Bromite, you should post on the Chromium developers mailing list.

**Reply by lordarcadius** [reply]
I asked here because Bromite doesn't show Top sites option by default.

**Reply by csagan5** [reply]
All the tiles on Bromite's NTP are the top sites.

**Reply by lordarcadius** [reply]
By Top Sites, I meant the Top Sites tile that only shows when you enable "explore-sites" flag in Bromite. But in case of Chromium, it even shows if the flag is set to default. So I just wanted to know how do I change the default as disable for this flag. I couldn't find anything in your patches
